src/cobra/manipulation/delete.py

```diff
@@ -185,62 +108,39 @@ def delete_model_genes(
         The list of genes to knock-out.
     cumulative_deletions: bool, optional
         If True, then any previous deletions will be maintained in the
-        model (default True).
+        model (default True). Unused, ignored.
```

Comment:
Any reason for adding this?

Reply:
I think I found a deprecation notice in the docstring, during my review. Was a long review so don't remember clearly. If the notice is there, you can remove this.

Reply:
Let's leave this PR alone for now. If you want to review #1208, I will merge all changes of it into the tests. Otherwise, I'm not going to change it here.